modules/core/src/smithy4s/schema/SchemaVisitor.scala

```diff
@@ -42,4 +42,27 @@ trait SchemaVisitor[F[_]] extends (Schema ~> F) {
     case SurjectionSchema(schema, to, from) => surject(schema, to, from)
     case LazySchema(make) => lazily(make)
   }
+
+  def mapK[G[_]](f: F ~> G): SchemaVisitor[G] = {
```

Comment:
To remove. `Schema ~> F` already has `andThen` which achieves exactly the same thing. 

https://github.com/disneystreaming/smithy4s/blob/main/modules/core/src/smithy4s/PolyFunction.scala#L28-L31

Reply:
Addressed in [73ec53e](https://github.com/disneystreaming/smithy4s/pull/295/commits/73ec53ebac13c50817bb5f799584856b840199d5)